frontend/src/metabase/public/components/widgets/SharingPane.styled.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import { space } from "metabase/styled-components/theme";
+
+export const IconContainer = styled.div`
+  width: 100px;
+  height: 63px;
+  flex-shrink: 0;
```

Comment:
This line fixed unaligned sharing options

#### Before
![image](https://user-images.githubusercontent.com/1937582/176402244-eb07322d-85ba-4044-b407-3afede7e9550.png)


#### After
![image](https://user-images.githubusercontent.com/1937582/176401909-b7d93955-8283-4626-8d66-ba314e92ac03.png)